book/chapter-5/README.md

```diff
@@ -1,6 +1,8 @@
-# Controlled and uncontrolled inputs
+# อินพุตควบคุม (Controlled Input) และ อินพุตอิสระ (Uncontrolled Input)
 
-These two terms *controlled* and *uncontrolled* are very often used in the context of forms management. *controlled* input is an input that gets its value from a single source of truth. For example the `App` component below has a single `<input>` field which is *controlled*:
+*อินพุตควบคุม (Controlled Input)* และ *อินพุตอิสระ (Uncontrolled Input)* จะถูกนำไปใช้ในการจัดการข้อมูล หรือ action ต่างๆของ form
+
+*อินพุตควบคุม (Controlled Input)* นั้นค่าของอินพุตจะถูกกำหนดด้วยข้อมูลจากภายนอก ที่เรามักจะใช้ค่านี้จากแหล่งข้อมูลที่หนึ่งที่มีค่าความจริงเพียงหนึ่งเดียวเท่านั้น (Single source of thruth) ดังตัวอย่างด้านล่าง component `App` มี element `<input>` อยู่หนึ่งตัว ซึ่งเป็น *Controlled Input* 
```

Comment:
>แหล่งข้อมูลที่หนึ่งที่มีค่าความจริงเพียงหนึ่งเดียวเท่านั้น (Single source of ~~thruth~~)

-> Single source of truth มันเป็น Design Pattern ในการเก็บข้อมูลแบบนึงอ่ะครับ อารมณ์ประมาณว่าทุกอย่างเก็บอยู่ในแหล่งข้อมูลเดียว ไม่ใช่แหล่งข้อมูลที่มีความจริงเพียงหนึ่งเดียวเท่านั้นครับ อันนั้นโคนัน 55555
สนใจอ่านเพิ่มจิ้มเบา ๆ ครับ [SSOT](https://en.wikipedia.org/wiki/Single_source_of_truth)
-> มี Typo ด้วยครับ "Truth"